packages/-ember-data/ember-cli-build.js

```diff
@@ -3,13 +3,16 @@
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
+  const isProd = process.env.EMBER_ENV === 'production';
+  const compatWith = process.env.EMBER_DATA_FULL_COMPAT ? '99.0' : null;
   let app = new EmberAddon(defaults, {
+    emberData: {
+      compatWith,
+    },
     babel: {
       // this ensures that the same `@ember-data/canary-features` processing that the various
-      // ember-data addons do is done in the dummy app
-      plugins: [
-        ...require('@ember-data/private-build-infra/src/debug-macros')(null, process.env.EMBER_ENV === 'production'),
-      ],
+      // ember-data addons do is done in the dummy app and tests
```

Comment:
don't think you added this, but the phrasing is pretty odd and I don't understand what the comment is saying

Reply:
I'll see if I can figure it out and improve it